provider/marathon/marathon_test.go

```diff
@@ -50,27 +51,12 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 		expectedBackends  map[string]*types.Backend
 	}{
 		{
-			desc: "simple application",
-			application: marathon.Application{
-				Ports:  []int{80},
-				Labels: &map[string]string{},
-			},
-			task: marathon.Task{
-				Host:  "localhost",
-				Ports: []int{80},
-				IPAddresses: []*marathon.IPAddress{
-					{
-						IPAddress: "127.0.0.1",
-						Protocol:  "tcp",
-					},
-				},
-			},
+			desc:        "simple application",
+			application: application(appPorts(80)),
```

Comment:
You seem to use `appPorts(80)` a lot, maybe it should be a default value in the application (and setting it only when you actually care about the value in your test) 👼